src/AutoCrudAdmin/Views/Shared/_EnityFormControlsPartial.cshtml

```diff
@@ -8,11 +10,75 @@
     .form-check-input {
         margin-left: 0.5rem;
     }
+    
+    ul.ui-autocomplete {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+        cursor: default;
+        
+        max-height:200px; 
+        max-width: 236px; 
+        
+        overflow-x:hidden;
+        overflow-y:scroll;
+    }
+    
+    .ui-helper-hidden-accessible {
+        display: none;
+    }
 </style>
 
 @foreach (var formControl in Model)
 {
-    <partial model="@formControl" name="@AutoCrudAdmin.Constants.PartialView.EntityFormControlPartial"/>
+    var autocompleteExists = Model.Count(fc => fc.DisplayName == formControl.DisplayName) > 1;
```

Comment:
I don't get how this indicates autocomplete exists?

Reply:
If we have an autocomplete form control, then we have one form control with the same display name that is automatically generated by the Autocrudadmin and is not of type autocomplete and one more that we are adding with type autocomplete. So if we have more than one form control with the same display name, one of them is definitely an autocomplete form control.

Reply:
Can you add a comment in the code about this, because I doubt I will be the only one wondering.

Reply:
Reworked as discussed IRL.